doc/manuals/admin/perf_tuning.md

```diff
@@ -187,8 +191,7 @@ Using both parameters, in any situation (either idle or busy) Orion consumes a f
 Apart from avoiding the thread exhaustion problem, there is a trade-off between using thread pools and
 not. On the one side, using thread pools is beneficial as it saves thread creation/destruction time.
 On the other hand, setting thread pools is a way of "capping" throughput. If the thread workers are busy
-all the time, at the end the queue saturates and you will end up losing incoming request or ongoing
-notifications.
+all the time, at the end the queue saturates and you will end up losing ongoing notifications.
```

Comment:
After last emails in MHD mailing list (http://lists.gnu.org/archive/html/libmicrohttpd/2016-12/msg00029.html) I understand that the "queue effect" only applies to our internal notification queue, so I have reomved "incoming request" from the text.